package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "svelte-app",
-  "version": "1.0.0",
+  "name": "ion_core",
+  "version": "2.2.0",
```

Comment:
Since we now have two places in which to change the version, we should probably:

- add a doc in the docs/ directory to document the release process;
- create a bash script to prepare the repo to cut a new release (i.e. bump the version in all the places we need to bump it in).

We did something similar in Glean: [script](https://github.com/mozilla/glean/blob/main/bin/prepare-release.sh), [docs](https://mozilla.github.io/glean/book/dev/cut-a-new-release.html).

Note that this is fine to do as a separate issue.

Reply:
We do have the start of docs for the first point https://github.com/mozilla-ion/ion-core-addon/blob/master/docs/ION_CORE_PUBLISHING.md

The second is a good idea, since we want to unblock this issue ASAP I'm going to file a separate issue for this as you suggest.

Reply:
> 
> 
> Since we now have two places in which to change the version, we should probably:
> 
>     * add a doc in the docs/ directory to document the release process;
> 
>     * create a bash script to prepare the repo to cut a new release (i.e. bump the version in all the places we need to bump it in).
> 
> 
> We did something similar in Glean: [script](https://github.com/mozilla/glean/blob/main/bin/prepare-release.sh), [docs](https://mozilla.github.io/glean/book/dev/cut-a-new-release.html).
> 
> Note that this is fine to do as a separate issue.

Filed issue #210